### ISSUE TITLE
Missing files on archive.org

### ISSUE THREAD

**Issue by markusmaresch** [gnosygnu/xowa#52]
Not sure about this, but it appears, that the front page of archive.org does not show ALL the files ..
(Close - if not an issue)

https://archive.org/details/Xowa_enwiki_latest ... incomplete list of files (for en_wiki 2016-03-05 files)
https://archive.org/download/Xowa_enwiki_latest ... (click on "Show all files") - here are the missing ones, they are hard to find - just look for 2016-03-05

Additional comment: Downloading large files by browser download is not very reliable.
Recommendation: Use: wget -cm https://archive.org/download/Xowa_enwiki_latest/Xowa_enwiki_2016-03-05_wiki_core.7z  ... this can be interrupted AND restarted - goes from there !!

**Comment by XowaWilliam** [comment]
Thanks Mark.
I can confirm that I opened a file called **err.txt** and logged there were files that I never downloaded, and are found in the link you gave: [](https://archive.org/download/Xowa_enwiki_latest)

Interestingly I never use **Xowa** to download because I have an unreliable service, and use FREE DOWNLOAD MANAGER because it handles interrupted downloads.

**Comment by XowaWilliam** [comment]
Woah... I just added up 14 files dated 2016-03-05 and it adds up to 50.7 GB
That's way over my monthly data bundle.
Mr Gnosy will have to confirm if all these files are compulsory. 

![wiki-files](https://cloud.githubusercontent.com/assets/12995082/14250849/8e91893a-fa80-11e5-9228-269c3aa96cc1.gif)

**Comment by gnosygnu** [comment]
@markusmaresch Thanks for opening the ticket. Yes, you are correct: There is a difference between these two urls:

* https://archive.org/details/Xowa_enwiki_latest: This is a manual list which I generate. It provides more info than the other one by archive.org. It also allows me to highlight specific files and "ignore" others.
* https://archive.org/download/Xowa_enwiki_latest: This is an automatic list which is generated by archive.org. It provides a full list of all XOWA files for English Wikipedia. It basically has three sets of files:
    * The files in my manual list
    * The files for Android. These are the "html" files. I'll explain more in my next comment
    * Old files from previous months. I usually keep old files around for a month or two before I delete them.

In short, I would use my manual list as it is shorter and provides more detail. But you can also pick whatever you want from the archive.org list.

> Downloading large files by browser download is not very reliable.
Recommendation: Use: wget -cm https://archive.org/download/Xowa_enwiki_latest/Xowa_enwiki_2016-03-05_wiki_core.7z 

I honestly use the default download manager in Firefox / Chrome. Every once in a while, I use a Firefox Addon called Down Them All, but it's been a while...

**Comment by gnosygnu** [comment]
@XowaWilliam Yup, there are 14 files dated 2016-03-05. They fall into three categories:

* Text-only databases for desktop XOWA: These are the files with "2016-03-05_wiki"
* Html databases for Android XOWA: These are the files with "2016-03-05_html"
* Image database updates for desktop / Android XOWA: This is the file with "2016-03-05_file"

With that said, keep the following in mind:
* If you're not using Android XOWA (i.e.: only using desktop XOWA), then you can skip the html files.
* If you're also downloading images, you should get the file database updates. Note that you have to get every "file_v2b" for previous months as well. For example, "Xowa_enwiki_2016-01-13_file_v2b.7z", "Xowa_enwiki_2016-02-03_file_v2b.7z"

Finally, sometime in the next month, I'm going to build a page in XOWA that will make these updates much easier. I know there are a lot of files to manage, and there's a lot of potential for confusion.

Let me know if there's anything else. Thanks.

**Comment by gnosygnu** [comment]
I updated archive.org tonight to point to all the 2016-03 versions. See: https://archive.org/download/Xowa_enwiki_latest

I'll mark this ticket closed next week unless there are any other questions / suggestions.

**Comment by gnosygnu** [comment]
As per above, this issue has been resolved. Thanks.
